src/groupByKey.ts

```diff
@@ -1,7 +1,18 @@
-type GroupsMap<T> = {
-  [key: string]: T[];
-};
-
-export function groupByKey(items, key) {
+export function groupByKey<T>(items: T[], key: keyof T): object {
   // write code here;
+  const result: any = {};
```

Comment:
```suggestion
  const result: GroupsMap<T> = {};
```
You had a type for it above. Use it
Also, it's better to use more descriptive names than just result

Reply:
Its works! Thanks.